packages/backend/src/routes/attestation.mjs

```diff
@@ -0,0 +1,13 @@
+import catchError from '../helpers/catchError.mjs'
+
+export default ({ app, db, synchronizer }) => {
+  const handler = async (req, res) => {
+    let attestation = await db.findOne('Attestation', {
+      where: {
+        attesterId: '1417799109672561583442883104695026698954826461290',
+      },
+    })
+    res.json(attestation)
```

Comment:
I haven't found a way to return ALL records of a table, is there one?
I was trying to get an array of all attestations (I think the original Ui design needed this?), also wanted all entries for other tables just bc I  was trying to pass ALL the data to the front end... so it may be the case that I'm trying to do something that isn't actually needed.

Reply:
Yep, you can do:

```js
const attestations = await db.findMany('Attestation', {
  where: {},
})
```

You can't load all the entries from all tables in a single call, but you can iterate over the table names and do the above.

```js
const allData = await Promise.all(Object.keys(db.schema).map(name => db.findMany(name, { where: {} })))
```
This will give you an array of arrays. But I'd caution against this, and instead suggest console logging or similar.